packages/thunderstore-api/src/client.ts

```diff
@@ -0,0 +1,35 @@
+import fetch from "node-fetch";
+
+class CommunityListResponse {
+  pagination: {
+    next_link: string | unknown;
```

Comment:
From Typescript's perspective, having `string | unknown` is the same thing as just having `unknown`. So at best this is type hinting for the human reader. Not sure about the background and aims of this ticket (the ticket itself is little help in the matter) so can't say if that's fine or not.

Reply:
The intention was to have the function return `knowntype | unknown` as that would force validation (or explicit ignorance of validation by casting) at the caller's end. The way it's implemented here does not really make sense though.